src/design-tokens/_scss-variables.scss

```diff
@@ -22,7 +22,7 @@ $cardWidthB3: 250px;
 $cardHeightB3: 375px;
 $cardGapB3: 20px;
 
-$cardWidthB4: 300px;
+$cardWidthB4: 210px;
```

Comment:
for this particular issue, i was thinking we set the content max with equal to 70vw and then have the cards automatically scale to fit that width with a gap.
the parent container should have display grid on it already, at the cards should be roughly 33% width, but i'm sure theres a responsive solution to not have to use arbitrary percentage values